source/fundamentals/data-formats/documents.txt

```diff
@@ -278,12 +283,77 @@ The output should resemble the following:
    title: Chronicle of a Death Foretold, yearPublished: 1981
    title: Love in the Time of Cholera, yearPublished: 1985
 
-For more information on retrieving and manipulating MongoDB data, see our
-:doc:`CRUD guide </fundamentals/crud>`.
-
 For more information on this class, see the
 :java-docs:`BsonDocument API documentation </apidocs/bson/org/bson/BsonDocument.html>`.
 
+JsonObject
+-------------
+
+The ``JsonObject`` class acts as a wrapper for JSON strings.
+If you only want to work with JSON data, you can use ``JsonObject``
+to avoid unnecessary data conversion to a ``Map`` object.
+
+By default, ``JsonObject`` stores :manual:`Extended JSON </reference/mongodb-extended-json/>`.
+You can customize the format of JSON in ``JsonObject`` by specifying a
```

Comment:
> by specifying

Consider showing how, since it's not straightforward: you need to create an instance of JsonObjectCodec and add it to a CodecRegistry.

Reply:
Thanks for the feedback Jeff. I added a code example section where we specify Relaxed mode JSON through a JsonObjectCodec: https://github.com/mongodb/docs-java/pull/57/commits/462dac7a56e1fdacb8954c065fcc5a844711a739

Let me know if this is inaccurate or unsatisfactory.